### PR TITLE
docs(src.html): keep monetization meta tag in built docs

### PR DESCRIPTION
**Pull request by sidvishnoi**

### CODE REVIEW

**Review by marcoscaceres — APPROVED** [review]
We should really control this through config... I'll have a look.

File: static/docs/src.html [hunk 1]
```diff
@@ -189,6 +189,11 @@
         }
       }
 
+      function keepMonetization() {
+        const meta = document.querySelector(`meta[name~="monetization"]`);
+        meta?.classList.remove("removeOnSave");
+      }
+
```
Comment (marcoscaceres) [inline static/docs/src.html:196]:
```suggestion
```

File: static/docs/src.html [hunk 2]
```diff
@@ -32,6 +32,7 @@
           addWikiLinks,
           fixMarkupPostprocess,
           postProcessEnhance,
+          keepMonetization,
```
Comment (marcoscaceres) [inline static/docs/src.html:35]:
```suggestion
```

File: static/docs/src.html [hunk 3]
```diff
@@ -32,6 +32,7 @@
...
           cleanup,
         ],
         logos: [
```
Comment (marcoscaceres) [inline static/docs/src.html:38]:
```suggestion
        monetization: {
          removeOnSave: false,
        },
        logos: [
```

**Comment by sidvishnoi** [comment]
@marcoscaceres Thanks for adding removeOnSave support!

**Comment by Psoffie009** [comment]
> @marcoscaceres Thanks for adding removeOnSave support!

**Comment by Psoffie009** [comment]
> We should really control this through config... I'll have a look.

**Comment by Psoffie009** [comment]
> @marcoscaceres Thanks for adding removeOnSave support!

**Comment by Psoffie009** [comment]
> We should really control this through config... I'll have a look.

**Comment by Psoffie009** [comment]
> We should really control this through config... I'll have a look.

**Pull request merged**